addons/payment_adyen/models/payment_transaction.py

```diff
@@ -282,3 +283,84 @@ def _adyen_tokenize_from_feedback_data(self, data):
         _logger.info(
             "created token with id %s for partner with id %s", token.id, self.partner_id.id
         )
+
+    def _adyen_extend_api_data(self, data):
+        self.ensure_one()
+        if 'sale_order_ids' not in self._fields:
+            # sale module is not installed
```

Comment:
You could just indeed check if the module is not installed; that'd be clearer.

Reply:
This requires extra query to db, no?

Reply:
Yes but I believe it would be trivial compared to the request.

Reply:
But my way seems even more trivial, no?

Reply:
Compute-wise, yes, clarity-wise, no. Both are fine anyway.